concent_api/concent_api/settings/testing.py

```diff
@@ -60,14 +60,14 @@
 
 STORAGE_CLUSTER_ADDRESS = 'http://localhost/'
 
-CONCENT_MESSAGING_TIME    = 2
-
 STORAGE_SERVER_INTERNAL_ADDRESS = 'http://localhost/'
 
 VERIFIER_STORAGE_PATH = '/tmp/'
 
 CONCENT_MESSAGING_TIME    = 0
 
+CONCENT_UPLOAD_RATE       = int(384 / 8)  # KB/s = kbps / 8
```

Comment:
Default value the same as in golem constants

Reply:
what about `development.py`?

Reply:
done